developer/config.yaml

```diff
@@ -9,10 +9,12 @@ cluster_type: openshift
 git_url: https://github.com/openshift-pipelines/pipeline-service.git
 
 # git_ref refers to the git repo's ref to be considered as the source of truth for Argo CD applications.
-git_ref: main
+git_ref: dev-kustomization
```

Comment:
Revert to `main`.

Reply:
Oh, nice catch thanks. I will fix it in all places.